routes/index.js

```diff
@@ -4,10 +4,11 @@ var Product = require('../models/product');
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
-  res.render('index', { products: /* ??? */ });
+  res.render('index', { products: Product.all});
```

Comment:
Looks good! This is the line of code you'll need to send the list of products to the page so they can be rendered.